ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -265,6 +265,58 @@ def test_tensorflow_eye(
     )
 
 
+@st.composite
+def elems_and_shape(draw):
```

Comment:
Is this helper necessary? Can we get a 1-D array of floats by utilizing already existing helpers?

Reply:
I resolved one of your comments, specifically the one about having the same signature of `tf.foldl()`. I am unsure if that was ok or not and could not revert it. :)